lib/galaxy/tools/error_reports/plugins/gitlab.py

```diff
@@ -199,6 +204,26 @@ def submit_report(self, dataset, job, tool, **kwargs):
             log.error("GitLab error reporting - No connection to GitLab. Cannot report error to GitLab.")
             return ('Internal Error.', 'danger')
 
+    def _open_issue(self, error_message, error_title, gitlab_projecturl, gl_project, gl_userid, issue_cache_key):
+        """ Open an issue """
+        import gitlab
```

Comment:
Can you pull the gitlab imports to the top and guard them like you do on line 48 ?

Reply:
I believe I have done so now